students/anbaopham/lesson09/donor_models.py

```diff
@@ -0,0 +1,86 @@
+import sys
+
+class DonorCollection(object):
```

Comment:
the relationship in DonorCollection and Donor classes isn't correct.

My suggestion is to review the instructions and tutorial for more help.

I also have this stub to demonstrate it more hands on:
https://repl.it/@naaleksandrova/Lesson9Example